README.md

```diff
@@ -24,3 +24,22 @@ Alternatively, if you are starting a fresh new library project, you use our defa
     
     # Runserver at port 8000
     make dev
+
+## For contributors
+
+### Requirements
+
+- Python 3
+- NodeJS
+
+### Build and Run
+
+1. Create a spinoff library
+
+        django-admin.py startproject --template=https://github.com/9gix/oils/zipball/project-template -n Makefile <project-name>
```

Comment:
Need to add README.md to the `-n` option, otherwise the `{{ project_name }}`s under README.md are not substituted